python/ray/serve/scripts.py

```diff
@@ -214,12 +248,16 @@ def run(config_or_import_path: str, address: str):
     try:
         # Check if path provided is for config or import
         is_config = pathlib.Path(config_or_import_path).is_file()
-
-        if address is not None:
-            ray.init(address=address, namespace="serve")
-        serve.start()
+        args, kwargs = process_args_and_kwargs(args_and_kwargs)
 
         if is_config:
+            if len(args) + len(kwargs) > 0:
+                raise ValueError(
```

Comment:
why not override here?

Reply:
By override, do you mean why we don't ignore `args` and `kwargs`? I thought it would be clearer to raise an error on unexpected input instead of silently ignore part of the command. It seemed less surprising to the user.